dependabot-core.gemspec

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+# This is a placeholder gem to prevent namesquatting on https://rubygems.org/gems/dependabot-core
+# Any updates must be manually published to RubyGems.
+# It's excluded from the normal release process because it's not expected to change.
+
+Gem::Specification.new do |spec|
+  spec.name = "dependabot-core"
+  spec.summary      = "This is a placeholder gem to prevent namesquatting. " \
+                      "You probably want the gem 'dependabot-omnibus'."
+  spec.description  = "This is a placeholder gem to prevent namesquatting. " \
+                      "You probably want the gem 'dependabot-omnibus'."
+  spec.authors     = "Dependabot"
+  spec.email       = "opensource@github.com"
+  spec.files       = [] # intentionally empty, this is a placeholder gem to prevent namesquatting
+  spec.homepage    = "https://github.com/dependabot/dependabot-core"
+  spec.license = "Nonstandard" # License Zero Prosperity Public License
+
+  spec.metadata = {
+    "bug_tracker_uri" => "https://github.com/dependabot/dependabot-core/issues"
+  }
+
+  spec.version = "0.95.1"
+  spec.required_ruby_version = ">= 3.1.0"
```

Comment:
It's dumb to include this `required_ruby_version`, but:
1. `rubocop` complains if I don't. 
2. So I tried setting it to `3`, but then `rubocop` complained that it wasn't as high as https://github.com/dependabot/dependabot-core/blob/17ab50090e14ffdf8efc3da01c0a4814d3d116dd/.ruby-version#L1
3. I tried disabling just that particular `rubocop` warning, but I couldn't figure out the magic incantation despite [checking the docs](https://docs.rubocop.org/rubocop/configuration.html#disabling-cops-within-source-code).

The unfortunate bit is that we'll end up having to bump this line _every_ time we bump `ruby`, which is trivial, but it makes the file look more active in `git blame` than it really is... and may lead to someone thinking they need to publish an updated version each time.

So I'd really rather figure out how to silence `rubocop` on this...

Reply:
Got it with a little more googling... basically I have to keep the key, but I can prevent it from needing to be bumped to match `.ruby-version`:
```ruby
spec.required_ruby_version = ">= 3" # rubocop:disable Gemspec/RequiredRubyVersion
```